Dataset/CY_Pilot/consumer_reports_metadata.json

```diff
@@ -0,0 +1,101 @@
+{
+  "id": "urn:ngsi-ld:Dataset:id:customer_reports:21675868",
+  "type": "Dataset",
+  "modified": "2024-04-09T14:09:32Z",
+  "source": [
+    "urn:ngsi-ld:Dataset:items:customer_reports:41266715"
+  ],
+  "description": [
+    "Reports by water consumers about issues they face related to their water. The dataset includes the reports, the assignment to operators and technical teams, as well as information about locations, type of problem, etc."
+  ],
+  "title": [
+    "Customer reports"
+  ],
+  "contactPoint": [
+    ""
+  ],
+  "belongsToCatalogue": "",
+  "distribution": [
+    ""
+  ],
+  "keyword": [
+    "water consumer",
+    "report",
+    "water issues",
+    "location",
+    "water problem type"
+  ],
+  "publisher": "WDME",
+  "spatial": [],
+  "temporal": [],
+  "theme": [
+    "Environment",
+    "Water"
+  ],
+  "accessRights": "",
+  "creator": [
+    "CY Pilot"
+  ],
+  "page": [
+    ""
+  ],
+  "accrualPeriodicity": "weekly",
+  "hasVersion": [
+    ""
+  ],
+  "identifier": [
+    ""
+  ],
+  "isReferencedBy": [
+    ""
+  ],
+  "isVersionOf": [
+    ""
+  ],
+  "landingPage": [
+    "https://dataportal-waterverse.opsi.lecce.it/dataset?organization=wbl"
+  ],
+  "language": [
+    "EN"
+  ],
+  "otherIdentifier": [
+    ""
+  ],
+  "provenance": [
+    ""
+  ],
+  "qualifiedAttribution": [
+    ""
+  ],
+  "qualifiedRelation": [
+    ""
+  ],
+  "relation": [
+    ""
+  ],
+  "issued": "1983-07-16T12:51:26Z",
+  "sample": [
+    ""
+  ],
+  "spatialResolutionInMeters": 0.0,
+  "temporalResolution": [
+    ""
+  ],
+  "Type": "",
+  "version": "",
+  "versionNotes": [],
+  "wasGeneratedBy": [
+    "wdme.com"
+  ],
+  "license": "Apache-2.0",
+  "geolocation": false,
+  "addressAvailable": false,
+  "standardization": "Level3Standardization",
+  "standardizationSource": "",
```

Comment:
If we have Level3 or Level4, then we need to provide the link to the standardization body (URL) here.

Reply:
Regarding this point, where can I find this standardization body (URL) ?

Reply:
It depends on each case, so if we have no details about the data that we have, I suggest to change to Level2Standardization due to in that case we do not need to put anything in standardizationSource.